src/modules/uv-shared-module/css/mixins-extended.less

```diff
@@ -51,7 +51,7 @@
 }
 
 .icon-btn-2(@img) {
-    background-image: data-uri(@img);
+    background-image: @img;
```

Comment:
why not use data-uri here?

Reply:
Webpack handles it automatically for us, so we can set a size limit in the webpack config and it will inline if its under a certain size, or copy it in and use a URL if its larger. I'm unsure what the default behaviour is.